docs/source/concept_models/vivarium_sanofi_multiple_myeloma/concept_model.rst

```diff
@@ -419,14 +419,109 @@ As discussed in treatment guideline reviews, in general triplet regimens are pre
      - High
      - VRd: bortezomib + lenalidomide + dexamethasone
 
-First-line treatment
-^^^^^^^^^^^^^^^^^^^^
+Treatment Modeling Strategy
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Second-line treatment
-^^^^^^^^^^^^^^^^^^^^
+**Scope**:
+
+This modeling strategy allows us to track the proportion of MM/RRMM simulants that receive an isatuxamib-containing treatment. It also allows for coherent assignment of isatuxamib-containing treatments at the simulant level so that any given simulant is not assigned an isatuxamib-containing treatment an illogical amount of times, which will reduce noise in our estimations.
+
+This model strategy does NOT consider:
+
+- Differences between isatuxamib-containing treatment regimens (ex: isatuxamib + pomalidomide + dexamethasone versus isatuxamib + carfilzomib + dexamethasone)
+- Differences between the numerous treatment regimens that do NOT contain isatuxamib (note this has important implications for applying efficacy data to the appropriate reference group(s))
+- Specific eligibility requirements for isatuxamib-contianing treatments other than previous treatment with isatuxamib. Specifically, 
+	- We do not directly consider the requirement of previous treatment with lenalidomide and a PI, although this is likely minimally impactful as these are very common treatment regimens and "almost all" patients with myeloma develop disease that is eventually refracotory to lenalidomide and PIs, [Attal-et-al-2019]_.
+	- We do not consider previous treatment with other monoclonal antibody drugs such as daratumumab, which may impact probability of receiving an isa-containing treatment
+
+We will consider the following treatment categories in our treatment model:
+
+- Isa-containing treatment regimen
+- Non-isa-containing treatment regimen
+
+.. note::
+
+	Other treatment categories to consider tracking include: reference treatment regimen from ISA RCT (polidamide + dexamethasone), daratumumab-containing regimens
+
+Importantly, we will assume that a given percentage of patients who receive isa-containing regimen for a specific line of treatment will not receive an isa-containing regimen at a later line of treatment.
```

Comment:
That is to say, Isa is a one-time treatment among myeloma patients. What about Dara and other types of regimen?

Reply:
It's a bit complicated... the recommendation is that at each line of treatment the patient should receive at least two drugs they have not previously received, preferable from another class (this is for all anti-myeloma drugs, not just isa). So previous treatment with dara/isa makes it less likely that a patient will be treated again with dara/isa, but not impossible. That's why I tried to structure it as a percent of patients, so that we could tweak this to 90% or some other value so that we allow for the possibility of retreatment after consultation with Manoj. Alternatively, I also suggested that we try to model a sort of refractory to isa model that would determine retreatment possibility (described in the to-do note)

Reply:
Thanks for explaining!